packages/crypto/src/__tests__/proof-tests.ts

```diff
@@ -0,0 +1,115 @@
+    deploymentLabel: "r2025.9.2",
+    enclaveApp: "signer",
+    owner: "tkhq",
+    createdAt: {
+      // 2025-09-16 21:24:01.448145+00
+      seconds: "1758057841",
+      nanos: "448145000",
+    },
+  };
+
+  test("should verify valid app proof signatures", () => {
+    expect(() => verifyAppProofSignature(testAppProof1)).not.toThrow();
+    expect(() => verifyAppProofSignature(testAppProof2)).not.toThrow();
+  });
+
+  test("should verify correct app proof / boot proof combos", async () => {
+    await expect(verify(testAppProof1, testBootProof1)).resolves.not.toThrow();
+    await expect(verify(testAppProof2, testBootProof2)).resolves.not.toThrow();
+  });
+
+  test("should NOT verify with malformed app proofs", () => {
+    let malformedAppProof2 = { ...testAppProof2 };
+
+    // Wrong publicKey
+    malformedAppProof2.publicKey = testAppProof1.publicKey;
+    expect(() => verifyAppProofSignature(malformedAppProof2)).toThrow();
```

Comment:
same thing here. I think we want to ensure:
1) the error messages are reasonable and easy to understand
2) the test is failing for the right reasons! (say this test failed with a good old "undefined is not a function" error...we don't want this test to pass!)